src/app/hooks/use-search-parameters-validation.ts

```diff
@@ -95,25 +119,16 @@ export const useSearchParametersValidation = () => {
 				throw new Error(t("TRANSACTION.VALIDATION.NETHASH_NOT_ENABLED", { nethash }));
 			}
 
-			const availableWallets = profile.wallets().findByCoinWithNethash(coin.toUpperCase(), nethash);
+			const availableWallets = profile.wallets().findByCoinWithNethash(coin, nethash);
 
 			if (availableWallets.length === 0) {
 				throw new Error(t("TRANSACTION.VALIDATION.NETHASH_NO_WALLETS", { nethash }));
 			}
 		}
 
```

Comment:
@dated shall we also include the method validation? Something like:
```suggestion
			if (!methods[method]) {
				throw new Error(t("TRANSACTION.VALIDATION.INVALID_METHOD"));
			}
```

Reply:
Its validated in a slightly different manner here: https://github.com/ArdentHQ/arkvault/pull/123/files#diff-e299cf03e5e8a2e1d02c779b9e96ea7f7b0a4305f9384f2056854356dc35ee4eR82